src/mlpack/core/optimizers/sgd/update_policies/momentum_update.hpp

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file momentum_update.hpp
+ * @author Arun Reddy
+ *
+ * Momentum update for SGD
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_OPTIMIZERS_SGD_MOMENTUM_UPDATE_HPP
+#define MLPACK_CORE_OPTIMIZERS_SGD_MOMENTUM_UPDATE_HPP
+
+namespace mlpack {
+namespace optimization {
+
+/**
+ * Momentum update policy for SGD.
+ *
+ */
```

Comment:
Could I ask you to please provide some more documentation on the momentum update policy (what it is, what it does, and how it works) in the comments for this class please?